lib/fake_idp/saml_response.rb

```diff
@@ -76,7 +97,7 @@ def replace_digest_value(document)
 
       # Replace digest node with the generated value
       document_copy = Nokogiri::XML(document_copy)
-      target_digest_node = document_copy.at_xpath("//ds:DigestValue")
+      target_digest_node = document_copy.at_xpath("//ds:DigestValue", "ds" => DSIG)
```

Comment:
What is "ds" and why is it used?

Reply:
Cool. Other than the `ruby-saml` gem, are there tools we can use to validate the syntax?

Reply:
I think Nokogiri or Xmlenc could be used to generally validate XML for syntax errors but I have a preference for `ruby-saml` in this use case since it's geared specifically towards enforcing details required by the SAML spec. In other words, we could produce a syntactically correct XML SAML document that may not be SAML spec compliant.

Reply:
@samudary Sorry I don't think I was clear... I meant as a sanity check. So, like copying and pasting the XML into some web-based linter to make sure `ruby-saml` isn't giving us some false-positive. This isn't a requirement, but rather a suggestion.

Reply:
@cbortz Gotcha! I've used both https://www.samltool.com/validate_response.php and https://www.samltool.com/prettyprint.php during development. I don't think I could've completed this work without them 😄 XML is unforgiving